packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx

```diff
@@ -1,222 +1,128 @@
-                                value={displayValue}
-                                style={{ marginBottom: 8 }}
-                                textColor={requiredEndColor}
-                            />
-                        );
-                    }
-                    return null;
-                })}
-                {(!!onStartButtonPress || !!onCompleteButtonPress) && (
-                    <View
-                        style={{
-                            gap: 16,
-                            flexDirection: "row",
-                            justifyContent: "center",
-                            marginTop: 22,
-                        }}
-                    >
-                        {onStartButtonPress && (
-                            <Button
-                                title={"Start"}
-                                onPress={onStartButtonPress}
-                                variant="outlined"
-                                disabled={isStartDisabled}
-                            />
-                        )}
-                        {onCompleteButtonPress && (
-                            <Button
-                                title={"Complete"}
-                                onPress={onCompleteButtonPress}
-                                variant="outlined"
-                                disabled={isCompleteDisabled}
-                            />
-                        )}
-                    </View>
-                )}
-            </View>
+                </View>
+            )}
+            {maintenanceType && <Label label={maintenanceType} style={styles.label} />}
+            <PropertyList
+                workOrder={rest}
+                overwriteLabel={overwriteLabel}
+                valueColor={valueColor}
+                currentDate={currentDate.toDate()}
+            />
+            {showActions && (
+                <View style={styles.actionContainer}>
+                    {onStartButtonPress && showActions.startButton && (
+                        <Button
+                            title="Start job"
+                            variant="outlined"
+                            disabled={isStartDisabled}
+                            onPress={onStartButtonPress}
+                        />
+                    )}
+                    {onCompleteButtonPress && showActions.completeButton && (
+                        <Button
+                            title="Ready for operation"
+                            variant="outlined"
+                            disabled={isCompleteDisabled}
+                            onPress={onCompleteButtonPress}
+                        />
+                    )}
+                    {onTecoButtonPress && showActions.tecoButton && (
+                        <Button
+                            title="Technical complete"
+                            variant="outlined"
+                            onPress={onTecoButtonPress}
+                        />
```

Comment:
Can all these buttons be available at the same time?

Also, since the buttons' visibility is dependent on `onStartButtonPress`, `onCompleteButtonPress`, and `onTecoButtonPress` respectively, is `showActions` necessary?

Reply:
Yes, however, there are specific conditions that must be met for these buttons to be displayed, which is the reason for the showActions prop.

Reply:
Yes, but the buttons' visibility are also dependent on `onStartButtonPress`, `onCompleteButtonPress`, and `onTecoButtonPress` respectively. To me that seems a bit redundant.

If a developer sets `showActions.startButton` to true, but `onStartButtonPress` is not defined yet, then the button will not be displayed. I would be confused if that happened to me

Reply:
ahh. I see